sys/include/net/netopt.h

```diff
@@ -213,6 +221,11 @@ typedef enum {
     NETOPT_ENCRYPTION,        /**< en/disable encryption */
     NETOPT_ENCRYPTION_KEY,    /**< set encryption key */
 
+    /**
+     * @brief get/set radio testmode e.g. for CE or FCC certification
+     */
```

Comment:
Even if not. All possible values should be documented here, otherwise noone will no what values are possible unless they look into the code, which sucks ;-)